eslint.config.mjs

```diff
@@ -1,13 +1,41 @@
 import globals from "globals";
-import tseslint from "typescript-eslint";
+import tseslint from "@typescript-eslint/eslint-plugin";
```

Comment:
As this would probably work, it's the old way of importing `tselint`. So unless there is a good reason, please stick to importing from `typescript-eslint`.